sw/device/tests/meson.build

```diff
@@ -1,5 +1,5 @@
 # Copyright lowRISC contributors.
-# Licensed under the Apache License, Version 2.0, see LICENSE for details.
```

Comment:
Don't remove lines from the licence header.

Reply:
Huh, good catch. I think that got clipped when I was moving stuff around.